docs/src/pages/guides/api/api.md

```diff
@@ -132,16 +132,16 @@ For example, let's take a button that has different types. Each option has its p
    and can easily support new values in the future.
 
 The Material-UI components use a combination of the two approaches according to the following rules:
-- A *boolean* is used when **2** degrees of freedom are required.
-- An *enum* is used when **> 2** degrees of freedom are required, or if there is the possibility that additional degrees of freedom may be required in the future.
+- A *boolean* is used when **2** possible values are required.
+- An *enum* is used when **> 2** possible values are required, or if there is the possibility that additional possible values may be required in the future.
 
-Going back to the previous button example; since it requires 3 degrees of freedom, we use an *enum*.
+Going back to the previous button example; since it requires 3 possible values, we use an *enum*.
 
 ### Ref
 
 The `ref` is forwarded to the root element. This means that, without changing the rendered root element
-via the `component` prop, it is forwarded to the outermost DOM element that which component
-renders. If you pass a different component via the `component` prop the ref will be attached
+via the `component` prop, it is forwarded to the outermost DOM element which the component
+renders. If you pass a different component via the `component` prop,docs: the ref will be attached
```

Comment:
```suggestion
renders. If you pass a different component via the `component` prop, the ref will be attached
```

Reply:
Yeah, sorry about that, lost focus from the GitHub commit message TextField :)